### PR TITLE
 HTML-24 - Make the navigations functionable 

### PR DESCRIPTION
**Pull request by hanii1993**
* Added navigations to main-page.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.container{ width:100%; margin:auto; }
+.header{ width:95%; margin:auto; }
+li{ list-style:none; margin: 10px 0}
+ul.navigations li{ list-style:none; display:inline-block; text-align:center; }
+nav > ul li > a{ padding:15px 100px; text-decoration:none; color:#fff; background:#267326; }
```
Comment (ban17) [inline style.css]: maybe selector can be `nav a`

**Review by ban17 — CHANGES_REQUESTED** [review]

File: main-page.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    let jsLists = document.getElementById('javascript-lists');
+    clickJS.addEventListener('click',openJS);
+
+    function openJS() {
+        jsLists.style.display = 'block';
+    }
+
+
+
+    var oddNumbers = document.getElementById('odd-numbers').addEventListener('click',oddNumberClick);
```
Comment (ban17) [inline main-page.html]: Use let or const, avoid var.

File: main-page.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    function oddNumberClick() {
+        var script = document.createElement('script');
+        script.type = 'text/javascript';
```
Comment (ban17) [inline main-page.html]: redundant - type text/javascript is default and not needed anymore

File: main-page.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        script.src = 'oddNumbers.js';
+        document.getElementsByTagName("head")[0].appendChild(script);
+        return false;
+    }
+
+</script>
+
+</body>
+</html>
```
Comment (ban17) [inline main-page.html]: set in editor to add new line always

File: main-page.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        <li><button>Form</button></li>
+        <li><button>Comments</button></li>
+    </ul>
+    <ul id="javascript-lists" class="languages-lists">
+        <li id="odd-numbers"><button>Odd Numbers</button></li>
+        <li id="even-numbers"><button>Even Numbers</button></li>
+        <li id="simple-prompt"><button>Simple Prompt</button></li>
+    </ul>
+
+
```
Comment (ban17) [inline main-page.html]: avoid adding blank lines randomly

File: main-page.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    </ul>
+
+    <ul id="html-lists" class="languages-lists">
+        <li><button>Calendar</button></li>
+        <li><button>Form</button></li>
+        <li><button>Comments</button></li>
+    </ul>
+    <ul id="javascript-lists" class="languages-lists">
+        <li id="odd-numbers"><button>Odd Numbers</button></li>
+        <li id="even-numbers"><button>Even Numbers</button></li>
```
Comment (ban17) [inline main-page.html]: Why this is button tag?

File: main-page.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    let cssLists = document.getElementById('css-lists');
+    clickCSS.addEventListener('click',openCSS);
+
+    function openCSS() {
+        cssLists.style.display = 'block';
+        return cssLists
+    }
+
+    let clickJS = document.getElementById('click-javascript');
+    let jsLists = document.getElementById('javascript-lists');
```
Comment (ban17) [inline main-page.html]: This need to be inside function openJS

File: main-page.html [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+    clickJS.addEventListener('click',openJS);
```
Comment (ban17) [inline main-page.html]: clickJS is used just once and can be written inline:
`document.getElementById('click-javascript').addEventListener('click',openJS);

File: main-page.html (hunk 2 above)
Comment (ban17) [reply main-page.html]: also remove `var oddNumbers` part it's not used

**Review by ban17 — CHANGES_REQUESTED** [review]

File: main-page.html [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+            <ul class="navigations">
+                <li><a href="#" id="click-html">HTML</a></li>
+                <li><a href="#" id="click-css">CSS</a></li>
+                <li><a href="#" id="click-javascript">JAVASCRIPT</a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <ul id="css-lists" class="languages-lists">
+        <li><a>CSS-grid</a></li>
```
Comment (ban17) [inline main-page.html]: Link must have href.

**Review by ban17 — CHANGES_REQUESTED** [review]
Add href to all links or make them normal text.

**Review by janko1031 — APPROVED** [review]
everything fine for me now.

File: main-page.html [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+    <ul id="html-lists" class="languages-lists">
+        <li><a href="#">Calendar</a></li>
+        <li><a href="#">Form</a></li>
+        <li><a href="#">Comments</a></li>
+    </ul>
+    <ul id="javascript-lists" class="languages-lists">
+        <li id="html-15"><a>HTML-15 - Odd Numbers</a></li>
+        <li id="html-16"><a>HTML-16 - Even Numbers</a></li>
+        <li id="html-17"><a>HTML-17 - Simple Prompt</a></li>
```
Comment (ban17) [inline main-page.html]: You should use href here too.

File: main.js [hunk 11]
```diff
@@ -0,0 +1,25 @@
+
+let clickHTML = document.getElementById('click-html');
+let htmlLists = document.getElementById('html-lists');
+clickHTML.addEventListener('click',openHTML);
+
+function openHTML() {
+    htmlLists.style.display = 'block';
+
```
Comment (ban17) [inline main.js:8]: remove redundant blank lines

File: style.css [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+.header{ width:95%; margin:auto; }
+li{ list-style:none; margin: 10px 0}
+ul.navigations li{ list-style:none; display:inline-block; text-align:center; }
+nav a{ padding:15px 100px; text-decoration:none; color:#fff; background:#267326; }
+nav a:hover{ background:#fff; color:#267326; border:1px solid #267326; }
+.kolona{ border:1px solid #ccc; width:30%; float:left; }
+.languages-lists{ display:none;}
+.languages-lists li{display:block; margin:25px 0}
+.languages-lists li>a{padding: 10px; margin: 10px 0; border: solid 1px #ccc; background: #eee; cursor:pointer;}
+
```
Comment (ban17) [inline style.css]: This line need to be empty (no spaces).

**Review by ban17 — CHANGES_REQUESTED** [review]

**Review by ban17 — APPROVED** [review]

**Pull request merged**